Sources/Controllers/Map/OAMapViewController.mm

```diff
@@ -1011,9 +1012,10 @@ - (void) storeTargetPosition:(UIGestureRecognizer *)recognizer
 
 - (void) restorePreviousTarget
 {
-    if ([self isTargetChanged] && [self isLastMultiGesture])
+    if ([self isTargetChanged] && ([self isLastMultiGesture] || _zoomInTargetChanged))
```

Comment:
Not clear why `_zoomInTargetChanged` needed. `isTargetChanged` should be enough.

Reply:
I added _zoomInTargetChanged to distinguish new restorePreviousTarget from the old one. Originally, it was designed to work during one gesture call, but now I save the target in one gesture and restore it in a different gesture call. So, isLastMultiGesture is NO when I call restorePreviousTarget.
As an alternative, I can create a separate restorePreviousTarget for this case.

Reply:
Now it is clear. Separate restorePreviousTarget not needed.